unlock-app/src/middlewares/walletMiddleware.js

```diff
@@ -225,6 +230,30 @@ const walletMiddleware = config => {
               }
             }
           )
+        } else if (action.type === SIGN_METADATA_REQUEST) {
+          const { address, owner, timestamp } = action
```

Comment:
this makes sense  would suggest changing `address` to `lockAddress` here so that its clear at first glance what this is